src/containers/Home.js

```diff
@@ -65,7 +68,13 @@ const Home = props => {
           <Typography gutterBottom variant='h6'>Bar Chart</Typography>
         </BarChartCard>
         <GroupedBarChartCard data={groupedBarChartData} feedbackId={'groupedBarChart'} classes={classes} />
-        {mapChartData ? heatmapData ? <MapChartCard data={mapData} classes={classes} /> : null : null}
+        {mapChartData ? heatmapData ? <MapChartCard
```

Comment:
We should figure out a better way to check that both data is loaded before rendering. It's pretty confusing with the nested ternary operators.

Reply:
For the world map, I think I can create data that has population and map info all in one json file to avoid this issue but yeah for future use cases where we grab data from 2 different api points, this would be helpful

Reply:
I tried implementing API caching here: https://github.com/justin0022/dashboard/blob/cache/src/hooks/useData.js

Got a weird error that said it's likely a bug with React, so I filed an issue here: https://github.com/facebook/react/issues/14979

There's some interesting development happening called [React Suspense](https://reactjs.org/docs/code-splitting.html#suspense) that may prove helpful for us, but [React Cache](https://www.npmjs.com/package/react-cache) is still in dev, so we'll have to implement a workaround for now.